### PR TITLE
Reimbursement steps

### PR DESCRIPTION
**Pull request by lwjohnst86**
Comments? Thoughts? Seems fairly simple I think...

### CODE REVIEW

**Comment by QuLogic** [comment]
You can't make a signed commit using the GitHub interface; then the private half of your GPG key wouldn't remain private ;)

**Comment by lwjohnst86** [comment]
oooo really?? Hmmm. OK, I'll change that

**Comment by lwjohnst86** [comment]
@mbonsma @linamnt Thoughts? I want to get consensus on this before merging in

**Comment by linamnt** [comment]
I think it looks good and simple enough!

**Comment by mbonsma** [comment]
How does the link to the PR work? I guess it's a minimum two-commit process then - you'd need to create the PR and then go back and add the link. Unless you know for sure which PR number you'll get, haha.

**Comment by mbonsma** [comment]
Yeah, it looks good!

**Comment by lwjohnst86** [comment]
@mbonsma that's exactly correct. I'll make that more explicit. 

Thanks everyone, I'll get this up and then you all can get your reimbursements! @linamnt You had to pay for the social room right? Can't remember.

File: treasurer/reimbursing/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+# Steps to get reimbursed
+
+1. Get your GitHub account [GPG verified](https://help.github.com/categories/gpg/) 
+(if you need help with that, come to a session and we'll get it set up)
+2. Fork the council repo and clone it to your computer, then follow these
+instructions:
+    - Copy `treasurer/reimbursing/template.md` to the `treasurer/reimbursing/submitted/` directory.
+    - Rename the file to `YYYY-MM-DD-firstname-lastname.md`: YYYY = year, MM =
+    month, DD = day (all numbers), and name = your name.
```
Comment (QuLogic) [inline treasurer/reimbursing/README.md]: Technically, there's no "name" in the templated filename.

File: treasurer/reimbursing/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    - Replace the relevant details to the file.
+    - Add and commit the file, push to your repo, and submit a pull request.
+    - Copy the pull request link to the file in the appropriate location.
+    - Commit and push the added change.
+3. (Optional) Scan your receipts, add them to the `receipts/` directory, and
+then (whenever you can). **(Required)** give the receipts to the treasurer.
```
Comment (QuLogic) [inline treasurer/reimbursing/README.md]: The first sentence ends without finishing.

File: treasurer/reimbursing/template.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- Name:
+
+## Receipt(s) list
+
+<!-- Replace the below example, following the format provided, and adding items as needed -->
+
+<!-- Example -->
+- Item 1:
+    - Date of purchase: 2016-04-01
+    - Description: Bulk barn foon
```
Comment (QuLogic) [inline treasurer/reimbursing/template.md]: foon?

**Comment by QuLogic** [comment]
You may want to mention that you need to do [git commit **-S**](https://help.github.com/articles/signing-commits-using-gpg/) and/or set `commit.gpgsign` to true so that the commit is signed.

**Comment by lwjohnst86** [comment]
Any final thoughts before I merge it in?

**Comment by QuLogic** [comment]
Now we just need to have a key signing party.

**Pull request merged**